kidviz/views.py

```diff
@@ -473,91 +475,57 @@ def selected_chart(self):
 
     def get_context_data(self, **kwargs):
         course_id = kwargs.get('course_id')
-
-        date_filtering_form = DateFilteringForm(self.request.GET, initial={
-            'course': self.request.GET.get('course', None),
-            'date_from': self.request.GET.get('date_from', None),
-            'date_to': self.request.GET.get('date_to', None),
-            'constructs': self.request.GET.get('constructs', None),
-            'tags': self.request.GET.get('tags', None) 
-        })
+        date_filtering_form = self._init_filter_form(self.request.GET)
 
         date_from = None
         date_to = None
         selected_constructs = None
         tags = None
+        time_window = WEEK_1
 
         if date_filtering_form.is_valid():
             date_from = date_filtering_form.cleaned_data['date_from']
             date_to = date_filtering_form.cleaned_data['date_to']
             selected_constructs = date_filtering_form.cleaned_data['constructs']
             tags = date_filtering_form.cleaned_data['tags']
+            time_window = date_filtering_form.cleaned_data['time_window']
```

Comment:
`time_window = date_filtering_form.cleaned_data['time_window'] or WEEK_1`
With such statement you dont need if from 494th line.